app-modules/ai/src/Jobs/CloneAiThread.php

```diff
@@ -82,6 +89,15 @@ public function handle(): void
         $threadReplica->locked_at = now();
         $threadReplica->save();
 
+        if (Feature::active('ai_utilization')) {
+            $this->thread->cloned_count = $this->thread->cloned_count + 1;
+            $this->thread->save();
+        }
+
+        if (Feature::active('ai_utilization') && Cache::has('special_actions_doughnut_chart')) {
+            Cache::forget('special_actions_doughnut_chart');
+        }
```

Comment:
No need to call the `Cache::has` here.

I think this one does make sense to leave here though.

Reply:
Done